app/javascript/components/enrollment/Enrollment.js

```diff
@@ -259,7 +259,7 @@ class Enrollment extends React.Component {
               jurisdiction_paths={this.props.jurisdiction_paths}
               assigned_users={this.props.assigned_users}
               first_positive_lab={this.props.first_positive_lab}
-              symptomatic_assessments_exist={this.props.symptomatic_assessments_exist}
+              symptomaticAssessmentsExist={this.props.symptomaticAssessmentsExist}
```

Comment:
why this change? props that come from the an erb file are usually done in snake case for us

Reply:
this prop is no longer used so this is no longer relevant: https://github.com/SaraAlert/SaraAlert/pull/861/commits/1670f66306ba72dce6246c84ead69a075c4a153f#diff-4db480ec24a44308ae9e46fd4ccfc147b86f1809d19ce73a86a78fab4d51d5d3L265